README.md

```diff
@@ -27,6 +27,19 @@ Next, add this to your crate:
 use bytes::{Bytes, BytesMut, Buf, BufMut};
 ```
 
+## no_std support
+
+To use `bytes` with no_std environment, disable the (enabled by default) `std` feature.
+
+```toml
+[dependencies]
+bytes = { version = "1", default-features = false }
+```
+
+To use `bytes` with no_std environment without atomic CAS, such as thumbv6m, you also need to enable
+the `extra-platforms` feature. See the [documentation for the `portable-atomic`
+crate](https://docs.rs/portable-atomic) for more information.
```

Comment:
It probably makes sense to say that this feature is not subject to the MSRV.

Reply:
Added the mention about the MSRV in https://github.com/tokio-rs/bytes/pull/467/commits/b30fe89f5212a6c54ecbffab17021e2ca538c653.

> The MSRV when `extra-platforms` feature is enabled depends on the MSRV of `portable-atomic`.